src/garage/np/algos/batch_polopt.py

```diff
@@ -22,17 +22,34 @@ class BatchPolopt(RLAlgorithm):
         baseline (garage.tf.baselines.Baseline): The baseline.
         discount (float): Discount.
         max_path_length (int): Maximum length of a single rollout.
+        n_samples (int): Number of policies sampled in one epoch.
```

Comment:
That's a pretty confusing explanation for `n_samples`. A better one might be "number of train_once calls per epoch" or something.